File: Source/ProspectingProject/Scripts/Core/ListingsManager/Models/NewHolidayListingModel.js



app.createHolidayListingModel = function () {
    var baseModel = new NewListingModelBase().create();
    return Object.assign(baseModel, {
        property_type: $("#propertyTypeInput option:selected").text(),
        establishment_name: $("#establishmentNameInput").val(),
        establishment_notes: $("#establishmentNotesInput").val(),
        bedrooms: $("#bedroomsInput option:selected").text(),
        sleeps: $("#sleepsInput option:selected").text(),

        king_beds: $("#kingBedsInput option:selected").text(),
        queen_beds: $("#queenBedsInput option:selected").text(),
        double_beds: $("#doubleBedsInput option:selected").text(),
        bunk_beds: $("#bunkBedsInput option:selected").text(),
        single_beds: $("#singleBedsInput option:selected").text(),
        bathrooms: $("#bathroomsInput option:selected").text(),
        baths: $("#bathsInput option:selected").text(),
        showers: $("#showersInput option:selected").text(),
        hand_showers: $("#handShowersInput option:selected").text(),
        lounge: $("#loungesInput option:selected").text(),
        dining_room: $("#diningRoomsInput option:selected").text(),
        garden: $("#gardenInput option:selected").text(),
        braai: $("#braaiInput option:selected").text(),
        pool: $("#poolInput option:selected").text(),
        jacuzzi: $("#jacuzziInput option:selected").text(),
        undercover_parking_bays: $("#undercoverParkingBaysInput option:selected").text(),
        open_parking_bays: $("#openParkingBaysInput option:selected").text(),
        peak_season: $("#peakSeasonInput").val(),
        semi_season: $("#semiSeasonInput").val(),
        low_season: $("#lowSeasonInput").val(),
        out_of_season: $("#outOfSeasonInput").val(),
        floor: $("#floorInput option:selected").text(),
        sea_views: $("#seaViewsInput option:selected").text(),
        cleaning_service: $("#cleaningServiceInput option:selected").text()
    });
}

File: Source/ProspectingProject/Controllers/Models/NewHolidayListingModel.cs
namespace ProspectingProject.Controllers.Models
{
    public class NewHolidayListingModel : ListingBaseModel
    {
        public string property_type { get; set; }
        public string establishment_name { get; set; }
        public string establishment_notes { get; set; }
        public string bedrooms { get; set; }
        public string sleeps { get; set; }
        public string king_beds { get; set; }
        public string queen_beds { get; set; }
        public string double_beds { get; set; }
        public string bunk_beds { get; set; }
        public string single_beds { get; set; }
        public string bathrooms { get; set; }
        public int baths { get; set; }
        public int showers { get; set; }
        public int hand_showers { get; set; }
        public string lounge { get; set; }
        public string dining_room { get; set; }
        public string garden { get; set; }
        public string braai { get; set; }
        public string pool { get; set; }
        public string jacuzzi { get; set; }
        public int undercover_parking_bays { get; set; }
        public int open_parking_bays { get; set; }
        public int peak_season { get; set; }
        public int semi_season { get; set; }
        public int low_season { get; set; }
        public int out_of_season { get; set; }
        public string floor { get; set; }
        public string sea_views { get; set; }
        public string cleaning_service { get; set; }
    }
}